src/Libraries/DynamoUnits/Quantity.cs

```diff
@@ -12,9 +10,9 @@ namespace DynamoUnits
     /// </summary>
     public class Quantity
     {
-        internal readonly ForgeUnitsCLR.Quantity forgeQuantity;
```

Comment:
did this namespace change? If so why is this package not version 5?

Reply:
answered in the general chat